test-framework/jacoco/runtime/src/main/java/io/quarkus/jacoco/runtime/ReportCreator.java

```diff
@@ -111,12 +97,23 @@ public void run() {
                 PrintStream ps = new PrintStream(out);
                 ps.println("Failed to generate Jacoco reports");
                 e.printStackTrace(ps);
-            } catch (IOException iugnore) {
+            } catch (IOException ignore) {
                 //ignore
             }
         }
     }
 
+    private static boolean jacocoRunning() {
+        try {
+            ManagementFactory.getPlatformMBeanServer().getObjectInstance(new ObjectName("org.jacoco:type=Runtime"));
```

Comment:
Do we really need the file size check now?

Reply:
Yes, the chance that we miss the MBean being there is pretty high (shutdown-hooks run concurrently).